.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,222 @@
+# Contributing Guidelines
+
+## Decision Making
+
+1.  Before each meeting, anyone who wishes may sponsor a proposal by filing an
+issue in the GitHub repository tagged "proposal", at least 24 hours before the
+meeting. Follow the "Proposal Issue Template".
+
+2.  A quorum is established in a meeting if half or more of voting members are present.
+
+3.  Once a person has sponsored a proposal, they are responsible for it.
+    The group may not discuss or vote on the issue unless the sponsor or their delegate is present.
+    The sponsor is also responsible for presenting the item to the group.
+
+4.  After the sponsor presents the proposal, a "sense" vote is cast for the proposal prior to any discussion:
+    -   Who likes the proposal?
+    -   Who can live with the proposal?
+    -   Who disagrees with the proposal?
+
+5.  If all or most of the group likes or can live with the proposal, it is immediately moved to a formal vote with no further discussion.
+
+6.  If most of the group disagrees with the proposal, it is postponed for further rework by the sponsor.
+
+7.  If some members disagree they can briefly state their objections.
+    A timer is then set for a brief discussion moderated by the facilitator.
+    After 10 minutes or when no one has anything further to add (whichever comes first),
+    the facilitator calls for a yes-or-no vote on the question:
+    "Should we implement this decision over the stated objections?"
+    If a majority votes "yes" the proposal is implemented.
+    Otherwise, the proposal is returned to the sponsor for further work.
+
+## Building and Previewing
+
+**Note:** we are using `book` as the master (published) branch because of the
+way GitHub handles organization repositories, domain names, and the generation
+of the website.
+
+**Setup to locally preview the book**:
+
+1.  Install R (we recommend that you also install and use the RStudio IDE).
+
+1.  Open RStudio by clicking on the `merely-useful.github.io.Rproj` file (if not
+through RStudio, then open an R console in the location of the Merely Useful
+repo) and install the dependencies by typing in the console:
+
+    ```r
+    # Make sure you have remotes installed
+    # install.packages("remotes")
+    remotes::install_deps()
+    ```
+
+**Build and preview from the command line**:
+
+1.  `make html` and then open `_book/index.html`.
+1.  `make pdf` or `make epub` to build PDF and EPUB versions (also in the
+`_book` folder).
+
+Or via RStudio:
+
+1. When in the R Project (opened via the `.Rproj` file), use the key bindings `Ctrl-Shift-B` to build the `html` output.
+
+### Continuous Building using Travis CI
```

Comment:
Maybe move this section last since it is more of an FYI than something contributors will need to worry about? It could be its on CI section instead of under building.